### DISCUSSION TITLE
Silent fail when spaces are present in tags list —

### DISCUSSION THREAD

**Discussion by slam-campup** [gavinr/github-csv-tools#92] [Help]
I imported a csv with 131 rows. The issues field had spaces in them, e.g. `"Engagement, Plugin, slam"`

github-csv-tools ran as expected listing "Imported issue:" for each row present. 
Output ended with `Created 131 issues, and had 0 failures.` 

However, 0 issues were present in the Github Issues for that repo. 

I edited the file to remove all spaces between items and to remove any empty items e.g. `,,`. 
Re-ran import in verbose mode.
github-csv-tools ran as expected listing "Imported issue:" for each row present. 
`Created 131 issues, and had 0 failures.`

However, there are only 110 items in the Issue. Some issues silent failed and I don't know why.

**Comment by gavinr** [comment]
Please provide a replication case.

**Comment by JoshuaCWebDeveloper** [comment]
I also faced this issue, and was disappointed that the issue was closed, as I felt the question outlined the problem well.

Here is an example csv of the first problem:
```csv
"title","labels","body"
"SAM-104 Document current technical state of project","jira-import,jira-import-230123,task,","Body content"
```
Notice how there is an empty label, a trailing comma.

Here is an example csv of the second problem:
```csv
"title","labels","body"
"SAM-56 Build athena image in balenaCloud builder","jira-import,jira-import-230123,task,Setup CI/CD, Deployment, and Automated testing","Body content"
```

Notice how there are spaces after the commas preceding some of the labels.


Label syntax issues aside, I think the most obvious issue is that github-csv-tools is silently failing and telling the user that it successfully imported an issue when it in fact did no such thing. At the very least, I would expect better error reporting so I am aware that the import failed (ideally, it could detect these syntax issues before it even starts the import and fail early).

**Reply by gavinr** [reply]
Thanks for the clarification. I would accept a PR that fixes the "Spaces before labels" issue.
